### PR TITLE
Navigate user to the created folder

### PR DESCRIPTION
**Pull request by ankitgoyalgloballogic**
#REF: #MOBILEAPPS-1174

### CODE REVIEW

File: ContentApp/PresentationLayer/Components/ListComponent/ListComponentViewController.swift [hunk 1]
```diff
@@ -309,6 +308,18 @@ class ListComponentViewController: SystemThemableViewController {
         self.forceRefresh(with: indexPaths)
     }
     
+    func openFolderAfterCreate(for node: ListNode?) {
+        let isMove = appDelegate()?.isMoveFilesAndFolderFlow ?? false
```
Comment (gl-lovekesh) [inline ContentApp/PresentationLayer/Components/ListComponent/ListComponentViewController.swift:312]: Remove the check for isMove. Make it constant for both flows.

**Review by gl-lovekesh — APPROVED** [review]

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]
please remove all isMove checks.

File: ContentApp/PresentationLayer/Components/ListComponent/ListComponentViewController.swift (hunk 1 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/PresentationLayer/Components/ListComponent/ListComponentViewController.swift]: Done

**Comment by ankitgoyalgloballogic** [comment]
Navigate to the folder after create

**Pull request merged**